src/layers/conv.jl

```diff
@@ -480,6 +480,71 @@ end
 outdims(l::CrossCor, isize) =
   output_size(DenseConvDims(_paddims(isize, size(l.weight)), size(l.weight); stride = l.stride, padding = l.pad, dilation = l.dilation))
 
+"""
+    AdaptiveMaxPool(out)
+
+Adaptive Max pooling layer. `out` is the size of the dimension of the output.
+
+"""
+struct AdaptiveMaxPool{O}
+    out::NTuple{O,Int}
+end
+
+function AdaptiveMaxPool(out::NTuple{O,Int})
+  return AdaptiveMaxPool(out)
+end
```

Comment:
This constructor is created by default when you define the struct. So no need to write this.

Reply:
I forgot to delete it.